traitsui/qt4/text_editor.py

```diff
@@ -72,16 +70,19 @@ def init(self, parent):
         if factory.password:
             control.setEchoMode(QtGui.QLineEdit.Password)
 
-        if factory.auto_set and not factory.is_grid_cell:
-            if wtype == QtGui.QTextEdit:
-                control.textChanged.connect(self.update_object)
-            else:
-                control.textEdited.connect(self.update_object)
-
+        self._signals = []
```

Comment:
`_signals` should probably be declared as a trait (`List()` is probably good enough).

Reply:
Now knowing what sort of things traits List does, I am less keen about bringing in `TraitListObject` for this...

Reply:
Done declaring `_signals` as a private trait:  This is also done in `traitsui.editor.Editor`, this is good because (1) there is a natural place for documentation, (2) we can actually gain benefits from the validating `Tuple(Any, Callable)` and (3) avoid `HasPrivateTraits` returning `None` in the unexpected places.